common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -111,19 +118,6 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         }
     }
 
-    @Override
-    public void onSaveInstanceState(@NonNull Bundle outState) {
```

Comment:
do not delete this, and also save the cert challenge handler class name as well.